charts/trento-server/charts/trento-web/templates/ingress.yaml

```diff
@@ -21,6 +21,10 @@ metadata:
   {{- with .Values.ingress.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
+    {{- if eq $.Values.ingress.className "traefik" }}
```

Comment:
Kind of unrelated, but I'm starting to think that, if we state that Trento only requires `any other CNCF-certified Kubernetes running on x86_64 architecture`, we would need to start documenting better all the `traefik` assumptions.

Whereas the official docs seem to be the proper place, what do you think of adding a more practical thing, like adding a disclaimer or something in the `NOTES.txt` if the ingressClass is not Traefik?